cli/lib/tsd-jsdoc/publish.js

```diff
@@ -645,7 +645,9 @@ function handleFunction(element, parent, isConstructor) {
         insideClass = isClassLike(parent);
         if (insideClass) {
             write(element.access || "public", " ");
-            if (element.scope === "static")
+            if (element.virtual)
+                write("abstract ");
+            else if (element.scope === "static")
```

Comment:
Looking good, except that I don't know if jsdoc might mark something as virtual that is also static. Wdyt of checking for static first?